### PR TITLE
Start an interview and tests

### PR DESCRIPTION
**Pull request by vvs79**
code review:

- [ ] @DimaSmagulov 
- [ ] @krisberry 
- [x] @natalya5 
- [x] @oleg08 
- [x] @OrestF 
- [x] @rtershak 
- [ ] @CHILL99 

2nd lvl review:

- [x] @Kmalynss
- [ ] @serhiym 
- [ ] @ruba-ruba

### CODE REVIEW

**Comment by OrestF** [comment]
reviewed

**Comment by vvs79** [comment]
https://trello.com/c/VxoE83hC/50-as-an-interviewer-i-want-to-be-able-to-start-an-interview

**Comment by oleg08** [comment]
reviewed

**Comment by rtershak** [comment]
reviewed.

File: app/views/interviews/_form.html.slim [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.form-group.center
```
Comment (DimaSmagulov) [inline app/views/interviews/_form.html.slim]: pls style this form like in questions _form.html.slim

**Comment by natalya5** [comment]
reviewed

**Comment by Kmalynss** [comment]
gtg

**Pull request merged**